tests/test_uploads.py

```diff
@@ -22,6 +21,47 @@
 from webcompat import app
 
 
+# taken from https://github.com/srusskih/flask-uploads/blob/master/flaskext/uploads.py#L476 # nopep8
+class TestingFileStorage(FileStorage):
```

Comment:
This change isn't very interesting, it's just copying and pasting from Flask-Upload so we can ditch the dependency and have our tests still working.